### PR TITLE
OCRVS-4286: Update on Rejection modal

### PR DESCRIPTION
**Pull request by yeasinhossain-dsi**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/457

### CODE REVIEW

**Comment by Zangetsu101** [comment]
@jpye-finch @naftis Yeasin had to add a new type of button `secondary_negative` here. Could you please take a look at those parts and see if you want any changes there?

File: packages/components/src/Button/Button.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ type ButtonType =
   | 'tertiary'
   | 'positive'
   | 'negative'
+  | 'secondary_negative'
```
Comment (naftis) [inline packages/components/src/Button/Button.tsx:25]: All good for me, only 1 thing, change this to camelCase? We use camelCase also with colors in `packages/components/src/colors.ts`

Comment (jpye-finch) [reply packages/components/src/Button/Button.tsx:25]: yep nice one

Comment (naftis) [reply packages/components/src/Button/Button.tsx:25]: Please also add a change log entry about this to `releases.stories.mdx`. 

```
<ReleasePill date="May 2023" version="1.3.0" />

- ✨ Add a new button type `secondaryNegative` to [Button](?path=/docs/controls-button--button)
```

**Pull request merged**